content/en/docs/components/katib/hyperparameter.md

```diff
@@ -0,0 +1,436 @@
++++
+title = "Getting started with Katib"
+description = "How to set up Katib and run some hyperparameter tuning examples"
+weight = 20
+                    
++++
+
+This page gets you started with Katib. Follow this guide to perform any
+additional setup you may need, depending on your environment, and to run a few
+examples using the command line and the Katib user interface (UI).
+
+For an overview of the concepts around Katib and hyperparameter tuning, check the
+[introduction to Katib](/docs/components/katib/overview/).
+
+## Katib setup
+
+This section describes some configurations that you may need to add to your
+Kubernetes cluster, depending on the way you're using Kubeflow and Katib.
+
+<a id="katib-install"></a>
+
+### Installing Katib
+
+You can skip this step if you have already installed Kubeflow. Your Kubeflow
+deployment includes Katib.
+
+To install Katib as part of Kubeflow, follow the
+[Kubeflow installation guide](/docs/started/getting-started/).
+
+If you want to install Katib separately from Kubeflow, or to get a later version
+of Katib, run the following commands to install Katib directly from its
+repository on GitHub and deploy Katib to your cluster:
+
+```shell
+git clone https://github.com/kubeflow/katib
+make deploy
+```
+
+### Setting up persistent volumes
+
+If you used the [above script](#katib-install) to deploy Katib,
+you can skip this step. This script deploys PVC and PV on your cluster.
+
+You can skip this step if you're using Kubeflow on Google Kubernetes Engine
+(GKE) or if your Kubernetes cluster includes a StorageClass for dynamic volume
+provisioning. For more information, check the Kubernetes documentation on
+[dynamic provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/)
+and [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+
+If you're using Katib outside GKE and your cluster doesn't include a
+StorageClass for dynamic volume provisioning, you must create a persistent
+volume (PV) to bind to the persistent volume claim (PVC) required by Katib.
```

Comment:
I think it's

```suggestion
StorageClass for dynamic volume provisioning, you must create a PersistentVolume (PV) to bind to the PersistentVolumeClaim (PVC) required by Katib.
```

(Maybe)

Reply:
I will use PV and PVC, since we mentioned it above.